bleachbit/Command.py

```diff
@@ -122,10 +122,10 @@ def __init__(self, path, func, label):
             raise AssertionError('Expected MethodType but got %s' % type(func))
 
     def __str__(self):
+        path = ''
```

Comment:
This could be confusing: path vs self.path

Reply:
Good point. How about renaming `path` to perhaps `info` or something better?